params/config.go

```diff
@@ -189,12 +189,14 @@ type WakuV2Config struct {
 	CustomNodes map[string]string
 
 	// PeerExchange determines whether WakuV2 Peer Exchange is enabled or not
+	// Deprecated: will be calculated based on LightClient
 	PeerExchange bool
```

Comment:
Just a question - these deprecation notices will be removed in the future, but not here, as this requires some additional unrelated work, correct?